src/test/java/software/amazon/awssdk/crt/test/MqttClientConnectionFixture.java

```diff
@@ -45,10 +47,10 @@
     }
 }
 
-public class MqttClientConnectionFixture {
+public class MqttClientConnectionFixture extends CrtTestFixture {
 
-    @Rule
-    public TestRule timeout = new DisableOnDebug(new Timeout(60, TimeUnit.MINUTES));
+    //@Rule
```

Comment:
@bretambrose I don't know what to do about this. These rules don't exist on Android, or I am too stupid to find them.

Reply:
Dunno what they are either!